src/Bswup/Bit.Bswup/Scripts/bit-bswup.progress.ts

```
@@ -62,28 +86,30 @@
                         hideApp_ && appEl && (appEl.style.display = 'none');
                         bswupEl && (bswupEl.style.display = 'block');
 
-                        if (showAssets_ && assetsEl) {
+                        if (showAssets_ && assetsEl && data.asset) {
                             const li = document.createElement('li');
                             li.innerHTML = `${data.index}: <b>${data.asset.url}</b>: ${data.asset.hash}`
                             assetsEl.prepend(li);
-                        }
-                        const percent = Math.round(data.percent);
+                        }                        const percent = Math.round(data.percent);
```

Biome (2.4.16)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)